### PR TITLE
Convert to be more generic-lens friendly.  Simplify field names.  Rem…

### PR DESCRIPTION
**Pull request by newhoggy**
…ove unused dependencies.

### CODE REVIEW

File: antiope-dynamodb/src/Antiope/DynamoDB/Types.hs [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+module Antiope.DynamoDB.Types where
+
+import Data.String           (IsString)
+import Data.Text             (Text)
+import GHC.Generics
+import Network.AWS.Data.Text (FromText (..), ToText (..))
+
+newtype TableName = TableName
+  { text :: Text
```
Comment (AlexeyRaga) [inline antiope-dynamodb/src/Antiope/DynamoDB/Types.hs]: can we make it something more descriptive, like `unTableName` or `tableName`?
Do we even need it? `toText` gives you exactly this value anyway. I think we should treat this type as a value by itself, not trying to extract its internals.

File: antiope-sqs/src/Antiope/SQS/Types.hs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import Data.String           (IsString)
+import Data.Text             (Text)
+import GHC.Generics
+import Network.AWS.Data.Text (FromText (..), ToText (..))
+
+data SQSError = DeleteMessageBatchError
+  deriving (Eq, Show, Generic)
+
+newtype QueueUrl = QueueUrl
+  { text :: Text
```
Comment (AlexeyRaga) [inline antiope-sqs/src/Antiope/SQS/Types.hs]: Same comment as for `TableName`

File: antiope-dynamodb/src/Antiope/DynamoDB/Types.hs (hunk 1 above)
Comment (newhoggy) [reply antiope-dynamodb/src/Antiope/DynamoDB/Types.hs]: Perhaps we can drop the field altogether.

**Comment by AlexeyRaga** [comment]
👍

**Pull request merged**